### PR TITLE
Update changelog with 2.10.0 and 2.10.1

### PR DESCRIPTION
**Pull request by j82w**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## <a name="2.10.0"/> 2.10.0 ##
+
+* Add support for creating geometry collections
+* Add support to specify bounding box for geometry spatial index
+* Add EnableTcpConnectionEndpointRediscovery to ConnectionPolicy
+* Fixed RuntimeBinderException for various query operations
+* Fixed permission serialization to include the Token
+* Improve retry logic for transient region failures
```
Comment (kirankumarkolli) [inline changelog.md]: Can it be more specific on what scenarios are handled.

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+## <a name="2.10.0"/> 2.10.0 ##
+
+* Add support for creating geometry collections
```
Comment (kirankumarkolli) [inline changelog.md]: Is it not possible before? There are some existing types related to geometry what are they for

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
+* Add support to specify bounding box for geometry spatial index
+* Add EnableTcpConnectionEndpointRediscovery to ConnectionPolicy
+* Fixed RuntimeBinderException for various query operations
```
Comment (kirankumarkolli) [inline changelog.md]: On what scenarios this exception gets thrown?

File: changelog.md [hunk 4]
```diff
@@ -1,3 +1,12 @@
+## <a name="2.10.0"/> 2.10.0 ##
+
+* Add support for creating geometry collections
+* Add support to specify bounding box for geometry spatial index
+* Add EnableTcpConnectionEndpointRediscovery to ConnectionPolicy
```
Comment (kirankumarkolli) [inline changelog.md]: Lets also list details on usage guidance.

**Comment by kirankumarkolli** [comment]
There are some query fixes as well right. Don't they need any mention?

File: changelog.md (hunk 2 above)
Comment (LalithaMV) [reply changelog.md]: Geometry collections support is added only now. Spatial types are common to both Geometry and Geography. Geography collections were supported from the beginning, so types have been around too.

**Pull request merged**